Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1122,20 +1122,9 @@ def self.curl_openssl_and_deps
     end
 
     def audit_urls
-      urls = [url] + mirrors
-
-      curl_openssl_or_deps = ResourceAuditor.curl_openssl_and_deps.include?(owner.name)
```

Comment:
@MikeMcQuaid `curl_openssl_or_deps` gets called in line [1137](https://github.com/Homebrew/brew/blob/efd69b16ed3c29cc1f65b010e43515495257ff00/Library/Homebrew/dev-cmd/audit.rb#L1137) and throws an `undefined local variable or method` error.